programs/dynamic-bonding-curve/src/instructions/admin/ix_close_claim_protocol_fee_operator.rs

```diff
@@ -15,10 +15,7 @@ pub struct CloseClaimFeeOperatorCtx<'info> {
     #[account(mut)]
     pub rent_receiver: UncheckedAccount<'info>,
 
-    #[account(
-        constraint = assert_eq_admin(admin.key()) @ PoolError::InvalidAdmin,
-    )]
-    pub admin: Signer<'info>,
+    pub signer: Signer<'info>,
 }
 
 pub fn handle_close_claim_fee_operator(ctx: Context<CloseClaimFeeOperatorCtx>) -> Result<()> {
```

Comment:
Could we rename the function name `handle_close_claim_fee_operator` to `handle_close_claim_protocol_fee_operator`, and context `CloseClaimFeeOperatorCtx` to `CloseClaimProtocolFeeOperatorCtx`?